### PR TITLE
MODINV-503 Edit MARC Authorities via quickMARC | Update Inventory Authorities

### PR DESCRIPTION
**Pull request by Shans-Kaluhin**
### Purpose 
 - Extend QuickMarcKafkaHandler to update Inventory Authorities record
 - Send new event QM_INVENTORY_AUTHORITIES_UPDATED after record has been updated

### Learn
[MODINV-503](https://issues.folio.org/browse/MODINV-503)

### CODE REVIEW

File: src/main/java/org/folio/inventory/dataimport/consumers/QuickMarcKafkaHandler.java [hunk 1]
```diff
@@ -76,9 +80,11 @@ public QuickMarcKafkaHandler(Vertx vertx, Storage storage, int maxDistributionNu
     this.kafkaInternalCache = kafkaInternalCache;
     this.instanceUpdateDelegate = new InstanceUpdateDelegate(storage);
     this.holdingsUpdateDelegate = new HoldingsUpdateDelegate(storage);
+    this.authorityUpdateDelegate = new AuthorityUpdateDelegate(storage);
     this.precedingSucceedingTitlesHelper = precedingSucceedingTitlesHelper;
     createProducer(kafkaConfig, QM_INVENTORY_INSTANCE_UPDATED);
     createProducer(kafkaConfig, QM_INVENTORY_HOLDINGS_UPDATED);
+    createProducer(kafkaConfig, QM_INVENTORY_AUTHORITIES_UPDATED);
```
Comment (psmagin) [inline src/main/java/org/folio/inventory/dataimport/consumers/QuickMarcKafkaHandler.java:87]: Please rename to `QM_INVENTORY_AUTHORITY_UPDATED`

**Review by psmagin — CHANGES_REQUESTED** [review]

File: src/main/java/org/folio/inventory/storage/external/ExternalStorageModuleAuthorityRecordCollection.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+  private static final Logger LOGGER = LogManager.getLogger(ExternalStorageModuleAuthorityRecordCollection.class);
+
+  ExternalStorageModuleAuthorityRecordCollection(
+    String baseAddress,
+    String tenant,
+    String token,
+    HttpClient client) {
+
+    super(String.format("%s/%s", baseAddress, "holdings-storage/holdings"),
```
Comment (Shans-Kaluhin) [inline src/main/java/org/folio/inventory/storage/external/ExternalStorageModuleAuthorityRecordCollection.java:27]: Do we need to create endpoints for authority-storage?
Like this [holdings-storage API](https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/holdings-storage.html)

Comment (psmagin) [reply src/main/java/org/folio/inventory/storage/external/ExternalStorageModuleAuthorityRecordCollection.java:27]: Not now.

Comment (psmagin) [reply src/main/java/org/folio/inventory/storage/external/ExternalStorageModuleAuthorityRecordCollection.java:27]: https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/authorities.html
We have them already.

Comment (Shans-Kaluhin) [reply src/main/java/org/folio/inventory/storage/external/ExternalStorageModuleAuthorityRecordCollection.java:27]: We definitely need to rename it to authority-storage like others 😅

**Review by psmagin — APPROVED** [review]

**Review by VRohach — COMMENTED** [review]
@Shans-Kaluhin Please update NEWS.md-file.

**Review by natali-zaitseva — APPROVED** [review]

**Pull request merged**